src/webcomponents/job/job-monitor.js

```diff
@@ -42,214 +41,209 @@ export class JobMonitor extends LitElement {
-        this.requestUpdate();
-
-        this.dispatchEvent(new CustomEvent("route", {
-            detail: {
-                hash: "#job",
-                resource: "job",
-                query: {id: jobId}
-            },
-            bubbles: true, // this is necessary as the event is handled in iva-app
-            composed: true
-        }));
+    onRefresh() {
+        this.fetchLastJobs();
     }
 
-    forceRefresh(e) {
-        e.stopPropagation();
-        this.fetchLastJobs();
+    onJobTypeChange(event, newJobType) {
+        event.stopPropagation();
+        this._visibleJobsTypes = newJobType;
+        this.requestUpdate();
     }
 
-    toggleDropdown() {
-        this.dropdown = !this.dropdown;
+    renderJobsButtons() {
+        return Object.keys(this.JOBS_TYPES).map(type => html`
+            <button class="btn btn-sm ${type === this._visibleJobsTypes ? "btn-secondary" : "btn-outline-secondary"} flex-fill" @click="${e => this.onJobTypeChange(e, type)}">
+                <strong>${this.JOBS_TYPES[type].title}</strong>
+            </button>
+        `);
     }
 
-    getDefaultConfig() {
-        return {
-            limit: 10,
-            interval: 30000
-        };
+    renderVisibleJobsList() {
+        // Get the list of visible jobs with the selected type
+        const visibleJobs = this._jobs.filter(job => {
+            return this._visibleJobsTypes === "ALL" || this.JOBS_TYPES[this._visibleJobsTypes].jobsTypes.includes(job?.internal?.status?.id);
+        });
+        if (visibleJobs.length > 0) {
+            return visibleJobs.map(job => html`
+                <li>
+                    <a href="${this.getJobUrl(job.id)}" class="dropdown-item border-top">
+                        <div class="d-flex align-items-center overflow-hidden">
+                            <div class="flex-shrink-0 fs-2 rocket-${job?.internal?.status?.id ?? job?.internal?.status?.name ?? "default"}">
+                                <i class="text-secondary fas fa-rocket"></i>
+                            </div>
+                            <div class="flex-grow-1 ms-3">
+                                ${this._addedJobs.has(job.id) ? html`
+                                    <span class="badge bg-primary rounded-pill">NEW</span>
+                                ` : nothing}
+                                <div class="mt-0 text-truncate" style="max-width:275px">
+                                    ${job.id}
+                                </div>
+                                <small class="text-secondary">
+                                    <span>${job?.tool?.id}</span>
+                                    <div class="vr"></div>
+                                    ${moment(job.creationDate, "YYYYMMDDHHmmss").format("D MMM YYYY, h:mm:ss a")}
+                                </small>
+                                <div>
+                                    ${UtilsNew.renderHTML(UtilsNew.jobStatusFormatter(job?.internal?.status))}
```

Comment:
Two comments here:
1. If we decide to display all possible job status, some of them are missing in the `jobStatusFormatter()` and should be included.
2. What do you think about adding a margin after the icon in `jobStatusFormatter()`?